src/hub/vendor/customer.py

```diff
@@ -143,9 +143,41 @@ def run(self) -> None:
                 last4=last4,
                 charge=charge,
             )
+            # NOTE:
+            # Firefox's route has a particular casing requirement
+            # Salesforce's route doesn't care.
```

Comment:
Makes we should make them the same so there is less confusion when trying to cross correlate them.

Reply:
I disagree, FxA team is not bending on this and Basket team has everything working, so I feel we should leave as it is and continue forward.